test/transmuxer/transmuxer_integration.js

```diff
@@ -107,9 +107,9 @@ describe('Transmuxer Player', () => {
       // seconds, fail the test.
       await waiter.waitForMovementOrFailOnTimeout(video, 10);
 
-      // Play for 15 seconds, but stop early if the video ends.  If it takes
+      // Play for 10 seconds, but stop early if the video ends.  If it takes
```

Comment:
This says 10, but you set it to play for 8.

...Honestly, we probably don't need these comments to begin with. All they do is say what variables we are passing into the function. The function has a pretty self-explanatory name.

Reply:
Done!